docs/source/how_to/how_to_derivatives.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        fun=sphere,
</pre><p>Do we need to pass <code>fun</code> if we pass <code>fun_and_jac</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/optimagic/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Currently yes but I created an issue to remove this limitation.